odoo_module_migrate/__main__.py

```diff
@@ -101,6 +101,12 @@ def get_parser():
         " the changes. (using git add and git commit command)"
     )
 
+    # TODO: Move to `argparse.BooleanOptionalAction` once in Python 3.9+
+    main_parser.add_argument(
+        "-npc", "--no-pre-commit", dest="pre_commit", action="store_false",
+        help="Skip pre-commit execution",
```

Comment:
I'd say that it misses ``default=False`` as in ``--no-commit``. Otherwise you break CI that don't expect to run pre-commit here.

Reply:
Well, this was ported from #41, where it was suggested to leave it active as the default: https://github.com/OCA/odoo-module-migrator/pull/41#discussion_r534173904

As this is still under "quick" development, I guess it is normal that sometimes things will break... Maybe we can start being more careful and making releases for this kind of changes?

Reply:
``action='store_false', default=False`` for --no-pre-commit, because it has dest.

Reply:
Well, if the default were to be **not to run pre-commit**, it would rather be something like:
```python
    main_parser.add_argument(
        "-pc", "--pre-commit", dest="pre_commit", action="store_true",
        help="Run pre-commit",
```
However, IMHO it should be enabled by default, I don't know what others think. But if this is mostly for OCA modules migration, where the usual flow is to run it, I think it could be useful... For specific situations where you don't want it, you can still disable it...  

I understand a change like this can break CIs all of a sudden, so that's why I was suggesting doing different releases so that people can pin versions downstream and only update when expected.

Reply:
Yeah, using releasing could solve such kind of problems

Reply:
> `action='store_false', default=False` for --no-pre-commit, because it has inverted dest.

I like it more as it is (in case our intention is to leave it active by default). [I have a personal vendetta against negative booleans](https://www.recallstack.icu/en/2020/09/01/do-not-use-negative-booleans-please/).

Also, as you can see in the implementation, if there's no `.pre-commit-config.yaml` file in the repo, pre-commit will not run anyways, so I think "do it" is the best default here.